helm/methode-content-collection-mapper/values.yaml

```diff
@@ -13,5 +13,7 @@ env:
     opts: "-Xms128m -Xmx128m -XX:+UseG1GC -server"
   document_store_api_url: ""
 resources:
+  requests:
+    memory: 128Mi
   limits:
     memory: 320Mi
```

Comment:
I see 330 MB spikes in last 7 days for this app, this will easily get OOM like this. I suggest putting a bigger limit, like 512MB.

Reply:
done